GitUI/LeftPanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -245,7 +245,7 @@ private void contextMenu_Opening(object sender, CancelEventArgs e)
 
             if (hasSingleSelection && selectedLocalBranch?.Visible == true)
             {
-                contextMenu.AddUserScripts(runScriptToolStripMenuItem, _scriptRunner.Execute, UICommands);
+                contextMenu.AddUserScripts(runScriptToolStripMenuItem, this, UICommands);
```

Comment:
I assume this is because the other parameters have a longer name (Although `UICommand` is set to `serviceProvider`, so it is arguable whether this is self-documenting). However, it would help if you explained such obscure suggestions, instead of letting me waste 10 minutes trying to figure out why that specific parameter needs to be different from the others.

Reply:
`null`, `false`, `true` should always be explained. For `this`, it usually improves the readability - particularly when the type or interface is not clear, which is the case here.
(I do not want to reopen the discussion about UICommands being a IServiceProvider.)